examples/speech_to_speech2/english_alternate_agent.py

```diff
@@ -0,0 +1,74 @@
+from simuleval.utils import entrypoint
```

Comment:
Nit: instead of adding a new folder speech_to_speech2, could we put these files under `speech_to_speech`?  I actually think we don't need to include `output` and `wavs` files - those are generated by running `simuleval`

Reply:
Yes, of course. Moved the files and removed the output and wavs files